views/stories/add.hbs

```diff
@@ -0,0 +1,31 @@
+<h3>Add Story</h3>
```

Comment:
Add storybook page, on this page, you will write the title of your story, story status public or private, and body field you can write you article.

Reply:
ok good job